defaults/main.yml

```diff
@@ -9,6 +9,12 @@ jenkins_version: 2.52 # latest
 
 jenkins_volume: "/dev/xvdb"
 
+jenkins_use_static_ssh_key: false
+
+jenkins_user_private_ssh_key: ""
+
+jenkins_user_public_ssh_key: ""
```

Comment:
This will populated using environment based value from vault? What will be the value if key is missing in vault? None or ""?